src/ROUTES.js

```diff
@@ -96,4 +96,22 @@ export default {
             isParticipantsRoute: Boolean(lodashGet(pathSegments, 2)),
         };
     },
+
+    /**
+     * React Navigation is failing to parse urls with dot in the path segment.
+     * This method enable pasring of urls with dot in path Segment
+     * @param {String} path - Path for the config
+     * @returns {Object}
+     */
+    getEmailRouteLinkingConfig: path => ({
+        path,
+        parse: {
+            login: l => (l
+                ? decodeURIComponent(l).replace('@dot@', '.').replace(encodeURIComponent('@dot@'), '@dot@')
+                : undefined),
+        },
+        stringify: {
+            login: l => (l ? l.replace('@dot@', encodeURIComponent('@dot@')).replace('.', '@dot@') : undefined),
```

Comment:
The logic here is also confusing... 

```js
l
    .replace('@dot@', encodeURIComponent('@dot@'))
    .replace('.', '@dot@')
```

- We are replacing the `@dot@` with `%40dot%40`
- Then we replace all `.` with `@dot@`... 

Why?

Reply:
Maybe, I have answered it above. First replace will escape the existing `@dot@` from email and then we can use `@dot@` to convert `.`.